frontend/src/metabase/api/dashboard.ts

```diff
@@ -25,136 +26,156 @@ import {
 } from "./tags";
 
 export const dashboardApi = Api.injectEndpoints({
-  endpoints: builder => ({
-    listDashboards: builder.query<
-      ListDashboardsResponse,
-      ListDashboardsRequest | void
-    >({
-      query: params => ({
-        method: "GET",
-        url: "/api/dashboard",
-        params,
+  endpoints: builder => {
+    const generateDashboardUpdateKeyMutation = <
```

Comment:
nit: maybe `generateDashboardUpdateMutationByKey`?

Reply:
I'll use `property` instead of `key` here too. Maybe this name can be `generateDashboardPropertyUpdateMutation` or something